source/fundamentals/crud/read-operations/geo.txt

```diff
@@ -0,0 +1,247 @@
+      if err != nil {
+         panic(err)
+      }
+
+For more information on using indexes, visit the :ref:`golang-indexes` guide.
+
+Coordinates on a 2D Plane
+-------------------------
+
+You can store geospatial data using ``x`` and ``y`` coordinates on 
+a two-dimensional Euclidean plane. We refer to coordinates on a two-dimensional
+plane as "legacy coordinate pairs."
+
+Legacy coordinate pairs have the following structure:
+
+.. code-block:: json
+
+   "<field name>" : [ x, y ]
+
+Your field should contain an array of two values in which the first represents 
+the ``x`` axis value and the second represents the ``y`` axis value.
+
+Index
+~~~~~
+
+To query data stored as legacy coordinate pairs, you must add the field containing
+legacy coordinate pairs to  a ``2d`` index. The following snippet creates a
+``2d`` index on the ``coordinates`` field:
+
+.. code-block:: java
+
+   indexModel := mongo.IndexModel{
+      Keys:    bson.D{{"location.geo.coordinates", "2d"}},
+   }
+
+   name, err := coll.Indexes().CreateOne(context.TODO(), indexModel)
+   if err != nil {
+      panic(err)
+   }
+
+For more information on legacy coordinate pairs, see the
+:manual:`MongoDB server manual page on legacy coordinate pairs </geospatial-queries/#legacy-coordinate-pairs>`.
+
+.. tip:: Supported Operators
+
+   Spherical (``2dsphere``) and flat (``2d``) indexes support some, but
+   not all, of the same query operators. For a full list of operators
+   and their index compatibility, see the
+   :manual:`manual entry for geospatial queries </geospatial-queries/#geospatial-query-operators>`.
+
+   .. external resource
+
+Geospatial Queries
+------------------
+
+To perform a geospatial query, create a query filter with a field name and a geospatial
+query operator. You can specify additional options for certain geospatial query operators
+to limit the documents returned.
+
+Query Operators
+~~~~~~~~~~~~~~~
+
+To query your geospatial data, use one of the following query operators: 
+
+- ``$near``
+- ``$geoWithin``
+- ``$nearSphere``
+- ``$geoIntersects`` *requires a 2dsphere index*
+
+When using the ``$geoWithin`` operator, you can specify the following shape operators:
+
+- ``$box``
+- ``$polygon``
+- ``$center``
+- ``$centerSphere``
+
+When using the ``$near`` operator, you can specify the following distance operators:
+
+- ``$minDistance``
+- ``$maxDistance``
+
+For more information on geospatial query operators, see the
+:manual:`manual entry for geospatial queries </geospatial-queries/#geospatial-query-operators>`.
+
+.. external resource
+
+Examples
+--------
+
+The following examples use the MongoDB Atlas sample dataset. You can load sample datasets into 
+your database on the free tier of MongoDB Atlas by following the :atlas:`Get Started with Atlas Guide
+</getting-started/#atlas-getting-started>` or you can :guides:`import the sample dataset into a local MongoDB instance
+</server/import/>`.
```

Comment:
Issue: Line 188 refers to "sample datasets" (plural) but line 190 refers to "the sample dataset" (singular). 

```suggestion
The following examples use the MongoDB Atlas sample dataset. You can load the sample dataset into 
your database on the free tier of MongoDB Atlas by following the :atlas:`Get Started with Atlas Guide
</getting-started/#atlas-getting-started>` or you can :guides:`import it into a local MongoDB instance
</server/import/>`.
```